service-mesh/control-plane/base/resource-templates/oauth-configmap.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
Shouldn't this be a Secret? :question:

Reply:
@israel-hdez Yes, good catch. Coming!

Reply:
@israel-hdez addressed in e3c1680. Thank you!